### DISCUSSION TITLE
Override <space> key for a particular site

### DISCUSSION THREAD

**Discussion by limaceous-bushwhacker** [gdh1995/vimium-c#1059] [Q&A] [answered]
How do I override space key to run `f+h` (follow and click h link)?

EDIT: I want to use space to pause/play video on YouTube.

**Comment by limaceous-bushwhacker** [comment]
Ok, I found how to do that by using `map <space> LinkHints.click match=<CSS selectors>`. How can I apply this to only on YouTube?

**Comment by limaceous-bushwhacker** [comment, marked as answer]
Might have finally figured it out. Is that the best way to do it?
```
env yt host="https://www.youtube.com/watch"
map <space> runKey expect="yt:<v-spc>" keys="<v-space>"
map <v-space> scrollFullPageDown
map <v-spc> LinkHints.click match="<CSS selector>"
```

**Reply by gdh1995** [reply]
Congratulations! This is just the recommended mapping.
